src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -170,20 +170,44 @@ static ucs_status_t uct_rc_mlx5_iface_arm_cq(uct_ib_iface_t *ib_iface,
 #endif
 }
 
+static ucs_status_t
+uct_rc_mlx5_ep_handle_failure(uct_rc_mlx5_ep_t *ep, ucs_status_t status)
```

Comment:
shouldn't this be in rc_mlx5_ep.c?

Reply:
then it must be non-static.. This is not a fast path, but I don't see the reason to increase @PLT when we can avoid this.

Reply:
"don't see the reason to increase @plt when we can avoid this" -> might as well put all functions in one file..
if we decided to put all functions whose subject is "ep" to rc_mlx5_ep.c - should keep this order

Reply:
ok, will fix